test/integration_test_helper.rb

```diff
@@ -21,7 +21,15 @@
 
 Selenium::WebDriver::Chrome::Service.driver_path = ENV['TESTDRIVER_PATH'] || Foreman::Util.which('chromedriver', Rails.root.join('node_modules', '.bin'))
 
-javascript_driver = ENV.fetch("JS_TEST_DRIVER") { ENV['DEBUG_JS_TEST'] ? :selenium_chrome : :selenium_chrome_headless }.to_sym
+javascript_driver = ENV.fetch("JS_TEST_DRIVER") do
+  if ENV['SHOW_JS_LOG']
+    :logging_selenium_chrome
+  elsif ENV['DEBUG_JS_TEST']
+    :selenium_chrome
+  else
+    :selenium_chrome_headless
+  end
+end.to_sym
 
 def chrome_options
   options = Selenium::WebDriver::Chrome::Options.new
```

Comment:
Any thoughts on implementing it here?
```suggestion
  options = Selenium::WebDriver::Chrome::Options.new
  options.add_option("goog:loggingPrefs", {browser: 'ALL'})  if ENV['SHOW_JS_LOG']
```

I think that gives you the same result but with less code.

Reply:
I still have to deal with ` ShowMeTheCookies.register_adapter(:logging_selenium_chrome, ShowMeTheCookies::SeleniumChrome) ` and `Capybara.register_driver :logging_selenium_chrome`  so it seems like the same amount of code?

Reply:
The `selenium_chrome_headless` driver is registered by default (https://github.com/nruth/show_me_the_cookies/blob/0cdaf2e0b099b0e3168f47c398bbe787e4f5442d/lib/show_me_the_cookies.rb#L18-L23) so you wouldn't need to do that. That's my whole point: please don't create a new driver.

Reply:
Thanks, didnt realize it will work this simply, updated!